src/server/api/user.js

```diff
@@ -174,10 +174,10 @@ export const resolvers = {
   },
   User: {
     ...mapFieldsToModel(
-      ["id", "firstName", "lastName", "email", "cell", "assignedCell", "terms"],
+      ["id", "firstName", "lastName", "alias", "email", "cell", "assignedCell", "terms"],
       User
     ),
-    displayName: user => `${user.first_name} ${user.last_name}`,
+    displayName: user => `${user.first_name}${user.alias ? ` (${user.alias}) ` : " "}${user.last_name}`,
```

Comment:
Should this be first name or alias in this case?

Reply:
The code is doing something like `FirstName (alias) lasName` when alias is provided. If not, it will display `firstName lasName`. You can see that on this [print screen](https://user-images.githubusercontent.com/38737958/72227990-9e7fb400-3570-11ea-8617-049efca1f045.png)